composer.json

```diff
@@ -66,6 +67,8 @@
         }
     },
     "scripts": {
+        "tests": "@php ./vendor/bin/paratest -p auto --runner=WrapperRunner",
+        "integration-tests": "@php ./vendor/bin/paratest -p half -c phpunit-integration-legacy.xml --runner=WrapperRunner",
```

Comment:
@alongosz  `test` or `tests`? Do we have other repos using such alias yet? _(if so I should align with it)_.

I'm thinking of combining these into so it's just one, to simplify going from repo to repo _(just like `fix-cs` alias)_:
```
"test": [
"@php ./vendor/bin/paratest -p auto --runner=WrapperRunner",
"@php ./vendor/bin/paratest -p half -c phpunit-integration-legacy.xml --runner=WrapperRunner"
],
```

Reply:
Can alos backport the alias to 1.7, but w/o paratest _(requires PHP7)_ to align things across branches.